src/main/java/dao/EventDao.java

```diff
@@ -16,6 +16,14 @@
 	public EventDao(DaoFactory daoFactory) {
 		this.daoFactory = daoFactory;
 	}
+	
+	public void closeConnection (Connection connection) {
+		try {
+			connection.close();
+		} catch (SQLException e) {
+			e.printStackTrace();
+		}
+	}
```

Comment:
same as with resolve null, you can extend dao classes and have the superclass contain this method